chainer_chemistry/links/update/gin_update.py

```diff
@@ -0,0 +1,78 @@
+import chainer
+from chainer import functions as F
+from chainer import links as L
+
+import chainer_chemistry
+from chainer_chemistry.links.connection.graph_linear import GraphLinear
+
+
+class GINUpdate(chainer.Chain):
+    """GIN submodule for update part.
+
+    Simplest implementation of Graph Isomorphism Network (GIN):
+    2-layered MLP + ReLU
+    no learnble epsilon
+
+    Batch Normalization is not implemetned. instead we use droout
+
+    # TODO: implement Batch Normalization
+    # TODO: use GraphMLP instead of GraphLinears
```

Comment:
thank you, i can change it to use GraphMLP.

Reply:
m(_ _)m